internal/config/config.go

```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package config provides configuration for all other internal packages.
```

Comment:
// Package config defines configuration used by the CLI.

(How the config is constructed seems like an implementation detail)